src/main/java/org/influxdb/impl/InfluxDBImpl.java

```diff
@@ -202,7 +197,13 @@ public QueryResult query(final Query query, final TimeUnit timeUnit) {
 	@Override
 	public void createDatabase(final String name) {
 		Preconditions.checkArgument(!name.contains("-"), "Databasename cant contain -");
-		this.influxDBService.query(this.username, this.password, "CREATE DATABASE IF NOT EXISTS \"" + name + "\"");
+		String createDatabaseQueryString = String.format("CREATE DATABASE IF NOT EXISTS \"%s\"", name);
+		DefaultArtifactVersion versionCurrent = new DefaultArtifactVersion(this.version());
```

Comment:
Wouldnt it be sufficient to check for this.version().startsWith("0.") to check for influxdb version < 1.